### PR TITLE
secretsmanager failing integration test added

### PR DESCRIPTION
**Pull request by bogdan-copocean**
Secrets Manager integration tests for failures

### CODE REVIEW

**Comment by aspacca** [comment]
all good, we should add test cases for the following as well:
https://github.com/elastic/elastic-serverless-forwarder/blob/main/share/secretsmanager.py#L57
https://github.com/elastic/elastic-serverless-forwarder/blob/main/share/secretsmanager.py#L60
https://github.com/elastic/elastic-serverless-forwarder/blob/main/share/secretsmanager.py#L92
https://github.com/elastic/elastic-serverless-forwarder/blob/main/share/secretsmanager.py#L100

File: tests/handlers/aws/test_handler.py [hunk 1]
```diff
@@ -91,6 +157,132 @@ def test_lambda_handler_failure(self) -> None:
...
+                        id: "arn:aws:secretsmanager:eu-central-1:123-456-789:secret:sqs_secret"
+                        outputs:
+                          - type: "elasticsearch"
+                            args:
+                              elasticsearch_url: "arn:aws:secretsmanager:eu-central-1:123-456-789:secret:es_secrets"
+                              username: "arn:aws:secretsmanager:eu-central-1:123-456-789:secret:es_secrets:es_username"
+                              password: "arn:aws:secretsmanager:eu-central-1:123-456-789:secret:es_secrets:es_password"
+                              dataset: "redis.log"
+                              namespace: "default"
+                              tags:
```
Comment (aspacca) [inline tests/handlers/aws/test_handler.py:204]: please, remove `outputs.[].tags` everywhere, no need to add `inputs.[].tags`

**Review by aspacca — COMMENTED** [review]
see comments

**Comment by bogdan-copocean** [comment]
added the rest of the tests

**Review by aspacca — APPROVED** [review]

**Pull request merged**